src/example-types/facet.js

```diff
@@ -29,6 +39,21 @@ module.exports = {
             },
           },
           node.size !== 0 && { $limit: node.size || 10 },
+          _.get('lookup', node) && {
+            $lookup: {
+              from: _.get('lookup.collection', node),
+              as: 'labelData',
+              localField: '_id',
+              foreignField: _.get('lookup.foreignField', node),
+            },
+          },
+          _.get('lookup', node) && {
```

Comment:
Nitpick: You can do these two stages in the same conditional

Reply:
yeah, will update